proto/dd_metric.proto

```diff
@@ -13,6 +13,19 @@ message CommonMetadata {
 	string api_key = 6;
 }
 
+message Origin {
```

Comment:
Note to reviewers: This was a change to prove this approach will work for Vector. Currently the official public proto file does not contain these additions. But hopefully will soon.